lib/slack.ex

```diff
@@ -148,14 +148,19 @@ defmodule Slack do
 
       def websocket_handle({:text, message}, _con, %{slack: slack, state: state}) do
         message = prepare_message message
-        if Map.has_key?(message, :type) do
-          {:ok, slack} = handle_slack(message, slack)
-
-          try do
-            {:ok, state} = handle_message(message, slack, state)
-          rescue
-            e -> handle_exception(e)
-          end
+
+        {slack, state} = case Map.has_key?(message, :type) do
```

Comment:
Works for me. Amended the commit to keep the history cleaner. This probably needs some further refactoring, but this should at least get things compiling without warnings on 1.3.